index.d/fabric8.yml

```diff
@@ -20,3 +20,14 @@ Projects:
     desired-tag: 1.0.1
     notify-email: cp.build.inb@karan.org
     depends-on: centos/centos:latest
+
+  - id: 3
+    app-id: fabric8-hdd-supervisor
+    job-id: openshift-hdd-supervisor
```

Comment:
I assume the significance of this is just to name the Jenkins job that would be created post adding this.
Please correct me if I am wrong @mohammedzee1000

Reply:
The container is named as a combination of app-id, job-id and desired-tag. These 3 form a composite key. The id field's primary significance, is to act as a primary key.
The final container name will be app-id/job-id:desired-tag, whatever values you provide there.

Reply:
Ok